src/server/storage/verifier.js

```diff
@@ -103,7 +100,7 @@ class FacebookVerificationStrategy {
     }
 
     logger.info('FacebookVerifier result:', { emailVerified })
-    userRecord.isEmailConfirmed = userRecord.isEmailConfirmed || emailVerified
+    return { emailVerified, mobileVerified: false }
```

Comment:
should this return mobileVerified: false ?

Reply:
@omerzam yes - this is defined in the `verify`'s method contract. it should return `{ emailVerified: boolean, mobileVerified: boolean }` object